src/main/java/org/gridsuite/voltageinit/server/service/parameters/VoltageInitParametersService.java

```diff
@@ -82,115 +84,153 @@ private Map<String, VoltageLimitEntity> resolveVoltageLevelLimits(VoltageInitRun
     public OpenReacParameters buildOpenReacParameters(VoltageInitRunContext context, Network network) {
-        AtomicReference<Long> startTime = new AtomicReference<>();
-        startTime.set(System.nanoTime());
-
-        Optional<VoltageInitParametersEntity> voltageInitParametersEntity = Optional.empty();
-        if (context.getParametersUuid() != null) {
-            voltageInitParametersEntity = voltageInitParametersRepository.findById(context.getParametersUuid());
-        }
-
-        OpenReacParameters parameters = new OpenReacParameters();
-        List<VoltageLimitOverride> specificVoltageLimits = new ArrayList<>();
-        List<String> constantQGenerators = new ArrayList<>();
-        List<String> variableTwoWindingsTransformers = new ArrayList<>();
-        List<String> variableShuntCompensators = new ArrayList<>();
-
-        voltageInitParametersEntity.ifPresent(voltageInitParameters -> {
-            if (voltageInitParameters.getVoltageLimits() != null) {
-                Map<String, VoltageLimitEntity> voltageLevelDefaultLimits = resolveVoltageLevelLimits(context, voltageInitParameters.getVoltageLimits().stream()
-                    .filter(voltageLimit -> VoltageLimitParameterType.DEFAULT.equals(voltageLimit.getVoltageLimitParameterType()))
-                    .toList());
-
-                Map<String, VoltageLimitEntity> voltageLevelModificationLimits = resolveVoltageLevelLimits(context, voltageInitParameters.getVoltageLimits().stream()
-                    .filter(voltageLimit -> VoltageLimitParameterType.MODIFICATION.equals(voltageLimit.getVoltageLimitParameterType()))
-                    .toList());
-
-                network.getVoltageLevelStream()
-                    .filter(voltageLevel -> voltageLevelDefaultLimits.keySet().contains(voltageLevel.getId()) || voltageLevelModificationLimits.keySet().contains(voltageLevel.getId()))
-                    .forEach(voltageLevel -> fillSpecificVoltageLimits(specificVoltageLimits, voltageLevelModificationLimits, voltageLevelDefaultLimits, voltageLevel, context.getVoltageLevelsIdsRestricted()));
-            }
-
-            constantQGenerators.addAll(toEquipmentIdsList(context.getNetworkUuid(), context.getVariantId(), voltageInitParameters.getConstantQGenerators()));
-            variableTwoWindingsTransformers.addAll(toEquipmentIdsList(context.getNetworkUuid(), context.getVariantId(), voltageInitParameters.getVariableTwoWindingsTransformers()));
-            variableShuntCompensators.addAll(toEquipmentIdsList(context.getNetworkUuid(), context.getVariantId(), voltageInitParameters.getVariableShuntCompensators()));
-        });
-        parameters.addSpecificVoltageLimits(specificVoltageLimits)
-            .addConstantQGenerators(constantQGenerators)
-            .addVariableTwoWindingsTransformers(variableTwoWindingsTransformers)
-            .addVariableShuntCompensators(variableShuntCompensators);
+        final long startTime = System.nanoTime();
```

Comment:
Too many changes. Add the logs in the existing code without changing everything

Reply:
have revert majority of file

Reply:
revert this change too